operations/mimir-tests/test-autoscaling.jsonnet

```diff
@@ -34,4 +34,8 @@ mimir {
     // the KEDA threshold
     k.util.resourcesRequests(2, '3.2Gi') +
     k.util.resourcesLimits(null, '6Gi'),
+  ruler_querier_container+::
+    // Test a <1 non-integer CPU request, to verify that this gets converted into an integer for
+    // the KEDA threshold
+    k.util.resourcesRequests(0.2, '1Gi'),
```

Comment:
eg, does this still work if we supply "1" or "0.2" etc.

Reply:
Thanks @jhesketh - I've changed the CPU request into '0.2' along with a comment that we're testing using a string, to make sure it works.